changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.6.3-beta3": [
+      "[New] Display \"pull with rebase\" if a user has set this option in their git config - #6553 #3422",
```

Personal style guide suggestion - `Git` for the product, `git` for the command line interface:

```suggestion
      "[New] Display \"pull with rebase\" if a user has set this option in their Git config - #6553 #3422",
```